libraries/mbed/targets/hal/TARGET_NXP/TARGET_LPC176X/TARGET_XBED_LPC1768/PinNames.h

```diff
@@ -72,8 +72,9 @@ typedef enum {
     LED3 = P1_21,
     LED4 = P1_23,
 
-    USBTX = P0_2,
-    USBRX = P0_3,
+    //For USB UART
+    //USBTX = P0_2,
```

Comment:
This platform is special. It doesn't have the mbed interface, I use the DIPDAP board as the interface board to debug it, so I need to change its UART pin setting. This pull request is used to change the default UART pins from USB-UART to DIP 25/26. But I still keep the code here for users who want to change the pin back to USB-UART.